Practice3/Program.cs

```diff
@@ -1,15 +1,62 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Practice3
 {
     class Program
     {
-        static void Main(string[] args)
+        private static int LowerBorder { get; set; }
+        private static int UpperBorder { get; set; }
+        private static int SumOfNumbersDevidableOn3AndNotDevidableOn5 { get; set; }
+        public static void Main(string[] args)
         {
+            Console.WriteLine("Set the lower value of the range!");
+            LowerBorder = Validator.GetValidatedInput();
+
+            Console.WriteLine("\nSet the upper value of the range!");
+            UpperBorder = Validator.GetValidatedInput();
+
+            if (IsRangeGoodToCreateArray(LowerBorder, UpperBorder))
+            {
+                int[] newArray = (int[])GetArrayOfNaturalNumbers(LowerBorder, UpperBorder);
```

Comment:
you shouldn't change the type by (int[]) operation.
If you need this type, you should return it from the GetArrayOfNaturalNumbers method